.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "openQA Single Instance",
+    "image": "registry.opensuse.org/devel/openqa/containers15.5/openqa-single-instance",
+    "runArgs": [ "--privileged", "--device", "/dev/kvm", "--entrypoint", "bash" ],
+    "postCreateCommand": "chown root:kvm /dev/kvm",
+    "containerEnv": {
+      "VNCPORT_OFFSET": "100"
+    },
+    "postStartCommand": "/usr/share/openqa/script/openqa-bootstrap >/var/log/openqa-bootstrap.log 2>&1"
```

Comment:
Seems like this is also suffering from whatever the example distri is suffering from?

https://github.com/os-autoinst/os-autoinst-distri-example/pull/41#pullrequestreview-2095899409